pkg/util/decimal.go

```diff
@@ -0,0 +1,67 @@
+package util
```

Comment:
I'd rather move it all out of `util` along with `Fixed8` (which is not used a lot these days). Somewhere into `encoding/fixedn` probably.

Reply:
Fixed.